yaml/csi-driver/v1.1.0/nimble-csp.yaml

```diff
@@ -0,0 +1,59 @@
+# Configuration to deploy the HPE Nimble Storage CSP service
+#
+# example usage: kubectl create -f <this_file>
+
+---
+
+#######################################
+############  CSP Service  ############
+#######################################
+
+kind: Service
+apiVersion: v1
+metadata:
+  name: nimble-csp-svc
+  namespace: kube-system
+  labels:
+    app: nimble-csp-svc
+spec:
+  ports:
+  - port: 8080
+    protocol: TCP
+  selector:
+    app: nimble-csp
+
+---
+
+##########################################
+############  CSP Deployment  ############
+##########################################
+
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: csp-service
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: nimble-csp
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: nimble-csp
+    spec:
+      priorityClassName: system-cluster-critical
+      containers:
+        - name: nimble-csp
+          image: hpestorage/nimble-csp:v1.0.0
```

Comment:
Should this be 1.1.0 to match the path?  Or even "edge" for now until we spin a beta?

Reply:
Is it better to create edge directory with YAML files pointing to edge images? with 1.1.0 directory, users might think this is the latest release and use these YAML files instead.

Reply:
That works too.  We probably shouldn't create the 1.1.0 directory until we release it.

Reply:
updated the folder and tag to edge